app/assets/stylesheets/_global.scss

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
```

Comment:
I usually leave the font size here or set it to an explicit value (16px). The `font-size: 62.5%` feels a little bit dated (what if users have a different font size set?) but if you like it feel free to keep it!